packages/foreman/rubygem-ffi/rubygem-ffi.spec

```diff
@@ -14,7 +14,7 @@
 
 Name:           %{?scl_prefix}rubygem-%{gem_name}
```

Comment:
It surprises me this is moved from plugins to foreman. Looks like `foreman-proxy` requires `rb-inotify` which requires ffi. It looks like before we pulled in rubygem-ffi version 1.9.10 from EPEL meaning this will be a downgrade.